CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.110.0] - 2024.04.28
+
+### Added
+- [customer] add customer by Ref Api Call
+
```

Comment:
This file must be in another commit which includes only CHANGELOG.md and package.json
Commit message like "chore(package): version 3.110.0" ...

Reply:
Right, i'll change it :+1: